bin/m

```diff
@@ -578,15 +578,7 @@ install_bin() {
   else # enterprise version
     case $os in
       linux* )
-        # for linux fetch the rhel7 tarball by default
-        local dist=rhel70
-        # shadowing earlier $distro
-        for distro in $distros; do
-          if good "http://downloads.10gen.com/$os/mongodb-$os-$arch-enterprise-$distro-$version.tgz"; then
-            dist=$distro
-            break
-          fi
-        done
+	local tarball="mongodb-$os-$arch-enterprise-$distro-$version.tgz"
```

Comment:
This removes the version heuristics added in https://github.com/aheckmann/m/pull/40. I believe the correct fix is https://github.com/aheckmann/m/pull/61/commits/3698834962b82b51fa23d9023cadee9135ae339e which I made in https://github.com/aheckmann/m/pull/61.